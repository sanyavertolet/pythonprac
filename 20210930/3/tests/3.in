56,75,44
88,44,22
34,43,22
1,0,0
0,1,0
0,0,1
