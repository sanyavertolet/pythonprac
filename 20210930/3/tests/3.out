[56, 75, 44]
[88, 44, 22]
[34, 43, 22]
